settings.gradle

```diff
@@ -7,3 +7,4 @@ rootProject.name = "ComposeSamples"
 include ':ListDetail', ':CompanionPane', ':DualView', ':ExtendedCanvas', ':ComposeGallery',
         ':TwoPage', ':NavigationRail', ':DyAdd', ':DragAndDrop', ':VideoPlusChat'
 
+include ':Diary'
```

Comment:
Would you please move this up, along with the rest of modules?

Reply:
yep, moved it!